_docs-sources/guides/stay-up-to-date/patcher/index.md

```diff
@@ -0,0 +1,74 @@
+# Patcher
+
+## Installation
+
+To install Patcher, download the appropriate package for your system from the [releases](https://github.com/gruntwork-io/patcher-cli/releases) page.
+
+### For MacOS and Linux users
+
+Patcher runs in single binary called `patcher`. Make sure it's available within your `PATH`.
+
+1. After downloading Patcher, unzip the package (only MacOS releases are zipped). Patcher runs in a single binary.
+2. If necessary, rename the binary to `patcher`.
+```
+mv patcher_linux_amd64 patcher
+```
+3. Move `patcher` to a location available in your `PATH`.
+```
+echo $PATH
+mv patcher /usr/local/bin
+```
+
+## Before Running Patcher
+
+To fetch information from GitHub, Patcher requires a GitHub Personal Access Token, with the `repo` scope. Set the
```

Comment:
> GitHub Personal Access Token

Consider linking this to GitHub's docs about PATs.

Reply:
Good point, would https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token be enough?